src/services/system_service.ts

```diff
@@ -36,12 +36,17 @@ class SystemsService {
     });
   }
 
+  sortSystemsVersion(systems: System[]) {
+    systems.sort((a, b) => (a.version > b.version ? -1 : 1));
```

Comment:
This doesn't have to be solved now, but this sort order is a bit naive.  As an example, when I create a bunch of versions of a system, I ended up with a sort order like this:

```
9.9.10
10.92.0
10.910.8
10.9.8
10.9.17
```

Where the above (assuming it's supposed to go newest to oldest) should go:

```
10.910.8
10.92.0
10.9.17
10.9.8
9.9.10
```

You have this sufficiently compartmentalized that we can come back and improve the sorting algorithm at a later point, I just wanted to get it on your radar.